src/app/container/paramfiles/paramfiles.service.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { Tag } from './../../shared/swagger/model/tag';
```

Comment:
One thought, shouldn't tests belong in a specific directory rather than along side regular code like `src/app/container/paramfiles/paramfiles.component.ts`

Reply:
Angular guide suggests it this way, and Angular-CLI generates it this way.

https://angular.io/guide/styleguide#style-04-06